include/ignition/physics/SpecifyData.hh

```diff
@@ -48,15 +48,6 @@ namespace ignition
       /// \brief Default constructor
       public: ExpectData();
 
-      /// \brief Copy constructor.
-      /// The copy constructor of the base class, CompositeData, is called
-      /// before this copy constructor and it does the actual copying of the
-      /// data contained in the MapData object of `_other`. Thus, this copy
-      /// constructor simply calls the default constructor to initialize the
-      /// MapData iterator of this object to point the appropriate iterator in
-      /// the newly copied MapData.
-      public: ExpectData(const ExpectData &_other);
```

Comment:
This was added in https://github.com/ignitionrobotics/ign-physics/pull/77 to fix a crash. Does it not occur anymore when removing it?

Reply:
It does, I meant to open this as a draft so that I didn't lose my train of thought.

I believe implementing the assignment operator is the correct solution to suppressing the warning in this case.